happi/audit.py

```diff
@@ -158,49 +158,21 @@ def validate_device_class(self, item):
                 except ImportError as ex:
                     logger.warning(ex)
 
-    def validate_enforce(self, entry, item):
-        for (key, value), info in zip(item.items(), entry.entry_info):
-            # print(f'item_______ {value}')
-            # print(type(value))
-            # print(info.enforce)
-            pass
+    def validate_extra_attributes(self, item):
```

Comment:
This function will need to be updated, just realized that this will be valid only for `OphydItems` or `HappiItems`.... not necessarily for some other containers.

Reply:
I don't see any particular assumptions in this method that wouldn't work for arbitrary container definitions.

Reply:
Ok thank you, i need to revisit this one, it might have been related to the fact that I did not have `pcdsdevices` in my path...